test/integration/model/geography.test.js

```diff
@@ -175,6 +175,52 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       });
     });
 
+    if (current.dialect.name === 'postgres') {
+      describe('GEOGRAPHY(POLYGON, SRID)', () => {
```

Comment:
I don't see any code changes related to this - tests for something else?

Reply:
@janmeier Yes, I decided to add some tests to cover GEOGRAPHY types while I was at it because I wanted to increase coverage of the postgres dialect.  Hope this doesn't prevent anything.

Reply:
Not at all, just wanted to make sure I wasn't missing anything